requirements.txt

```diff
@@ -1,10 +1,15 @@
 networkx==2.5
```

Comment:
Why did you change the requirements?

Reply:
By accident, the pycharm showed me a message that the requirements file doesn't contain all the necessary requirements and I chose the option of solving it automatically.
I'll revert those changes, but nonetheless, shouldn't I include scipy in the requirements now? because I'm using it for the minimization.